source/Camera/Basic/Capture2D/Capture2D.cpp

```diff
@@ -20,10 +20,10 @@ int main()
         // Note: The Zivid SDK supports 2D captures with a single acquisition only
         const auto settings2D =
             Zivid::Settings2D{ Zivid::Settings2D::Acquisitions{ Zivid::Settings2D::Acquisition{
-                                   Zivid::Settings2D::Acquisition::ExposureTime{ std::chrono::microseconds{ 10000 } },
-                                   Zivid::Settings2D::Acquisition::Aperture{ 2.83 },
-                                   Zivid::Settings2D::Acquisition::Brightness{ 1.0 },
-                                   Zivid::Settings2D::Acquisition::Gain{ 1.0 } } },
+                                   Zivid::Settings2D::Acquisition::ExposureTime{ std::chrono::microseconds{ 30000 } },
+                                   Zivid::Settings2D::Acquisition::Aperture{ 11.31 },
+                                   Zivid::Settings2D::Acquisition::Brightness{ 1.80 },
+                                   Zivid::Settings2D::Acquisition::Gain{ 2.0 } } },
```

Comment:
The [Capture.cpp](https://github.com/zivid/zivid-cpp-samples/blob/64862fa74d67153a4e1eca4ca70bb64efbcd7cdd/source/Camera/Basic/Capture/Capture.cpp#L21) uses the default gain equal to 1.0. Why is it an improvement to use gain equal to 2.0 here?

Reply:
We want low iris (high f-number) to have the 2D image in focus. For that reason, we need to crank up other settings to get high enough exposure. Gain does not affect 2D images the same as it affects 3D images so there is no reason to be restrictive about using it.